grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/NullGrpcClientStubFactory.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.client.grpc;
+
+import javax.annotation.Nullable;
+
+import com.linecorp.armeria.client.grpc.GrpcClientStubFactory;
+
+import io.grpc.Channel;
+import io.grpc.ServiceDescriptor;
+
+public enum NullGrpcClientStubFactory implements GrpcClientStubFactory {
+
+    INSTANCE;
+
+    @Override
+    public ServiceDescriptor findServiceDescriptor(Class<?> clientType) {
+        return null;
```

Comment:
How about throwing an exception just in case this class is ever invoked by mistake?